functions/bouncer/src/index.js

```diff
@@ -0,0 +1,48 @@
+import 'dotenv/config';
+import express from 'express';
+import * as Sentry from '@sentry/node';
+import responseTime from 'response-time';
+import {
+  stripeEventParse,
+  stripeEventCsCompleted,
+  stripeEventCustCreated,
+  stripeEventQueue,
+  stripeEventEnd,
+  errorHandler,
+} from './middleware';
+
+const api = express();
+
+let version;
+(async () => {
+  let p;
+  try {
+    p = await import('./package.json');
+  } catch {
+    p = await import('../package.json');
+  }
+  version = p.version;
+})();
+const defaultVersion = `bouncer@${version}`;
+Sentry.init({
+  dsn: process.env.SENTRY_DSN,
+  environment: process.env.NODE_ENV,
+  release: process.env.SENTRY_VERSION || defaultVersion,
+  debug: process.env.NODE_ENV === 'development',
+});
+Sentry.configureScope(scope => {
+  scope.setTag('thatApp', 'bouncer');
+  scope.setTag('subSystem', 'checkout');
+});
+
+export const handler = api
```

Comment:
again should we be setting the firestorm instance here and passing it down?

Reply:
lets discuss approach please

Reply:
Refactored in bouncer: 8b14f8a